android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/Window.kt

```diff
@@ -38,6 +38,6 @@ fun Window.setNavigationBarTheme(@ColorInt toolbarColor: Int) {
 /**
  * Retrieves a {@link WindowInsetsControllerCompat} for the top-level window decor view.
  */
-fun Window.getWindowInsetsController(): WindowInsetsControllerCompat {
+fun Window.createWindowInsetsController(): WindowInsetsControllerCompat {
```

Comment:
nit: I don't think we should change this

Reply:
well, isn't `getWindowInsetsController` misleading? It wasn't a getter for a parameter, it was actually returning a new instance.